starter-project-backend/src/server.ts

```diff
@@ -1,18 +1,23 @@
-import app from "./app"
-import mongoose from 'mongoose';
+import app from "./app";
+import mongoose from "mongoose";
 
-const PORT = process.env.PORT || 8000
-const DB_URI = process.env.MONGO_URI || "";
+const PORT = process.env.PORT || 8000;
+const DB_URI = process.env.MONGO_URI || "mongodb://localhost:27017/";
 
 // @ts-ignore
-mongoose.connect(DB_URI, {
+mongoose
+  .connect(DB_URI, {
     useNewUrlParser: true,
     useUnifiedTopology: true,
     useCreateIndex: true,
     useFindAndModify: false,
-})
-.then(() => {
-    app.listen(PORT, () => console.log('Server running...'));
-})
-.catch((err: any) => console.log('Error occurred while connecting', err));
+    dbName: "esklatePractice",
+  })
+  // .then(() => {
+  //   app.listen(PORT, () => console.log("Server running..."));
+  // })
+  .catch((err: any) => console.log("Error occurred while connecting", err));
 
```

Comment:
better not to change this file, because it not convenient to use the actual atlas db